edalize/templates/yosys/yosys-script-tcl.j2

```diff
@@ -1,4 +1,5 @@
 yosys -import
+{{ plugins }}
```

Comment:
Loading of plugins needs to be before ``yosys -import``, so tcl script will have access to plugins commands

Reply:
good point, I have changed that to proper order.
I created a minimal test that checks if the uhdm plugin is generate corectly